### ISSUE TITLE
REST Api

### ISSUE THREAD

**Issue by superandrew** [llazzaro/django-scheduler#526]
Hi, is this package compatible with DRF? Is it supported to run calendar with no frontend UI (e.g. for a mobile client)?

**Comment by superandrew** [comment]
I noticed that there are API, however in the documentation (and the locally run demo) I wasn't able to find references to new event creation via API (I found the url, but I am unable to call it in POST with correct parameters). I'll try to reverse engineer the code, I was just guessing if I was missing a obvious thing in the documentation.

**Comment by xjlin0** [comment]
Yes I am using this package with DRF with home written endpoints. Also you can check how other people use it
https://github.com/llazzaro/django-scheduler/issues/304#issuecomment-341237923
